.commitlintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": ["@commitlint/config-conventional"],
+    "rules": {
+        "type-enum": [
+            2,
+            "always",
+            [
+                "build",
+                "busybot",
+                "chore",
+                "ci",
```

Comment:
Can you add `cd` as well, have been using it for some commits lately..

Reply:
What does it stand for?